packages/next-swc/package.json

```diff
@@ -9,6 +9,7 @@
     "clean": "node ../../scripts/rm.mjs native",
     "build-native": "napi build --platform -p next-swc-napi --cargo-cwd ../../ --cargo-name next_swc_napi --features plugin,image-extended --js false native",
     "build-native-release": "napi build --platform -p next-swc-napi --cargo-cwd ../../ --cargo-name next_swc_napi --release --features plugin,image-extended,tracing/release_max_level_info --js false native",
+    "build-native-release-with-assertions": "napi build --platform -p next-swc-napi --cargo-cwd ../../ --cargo-name next_swc_napi --cargo-flags=--profile=release-with-assertions --features plugin,image-extended,tracing/release_max_level_info --js false native",
```

Comment:
```suggestion
    "build-native-release-with-assertions": "napi build --platform -p next-swc-napi --cargo-cwd ../../ --cargo-name next_swc_napi --profile release-with-assertions --features plugin,image-extended,tracing/release_max_level_info --js false native",
```

Reply:
> [-p #0] 

Maybe `-p`?

Reply:
woo hoo, that worked!

i see the first debug assertion in ci
```
duplicate module idents in graph: [
      "[project]/app/base2-scss.module.scss.module.css [app-client] (css)",
  ]] {
  type: 'TurbopackInternalError'
}
```

Reply:
Yep, that is the one remaining problem listed in https://github.com/vercel/next.js/pull/78061

Reply:
ok, i will split this up!

PR1: update napi cli
PR2: try to enable debug_assertions for some of the ci builds
then you can put the cherry on top with that last fix